### ISSUE TITLE
Discrepancy on text selection when preview window is open

### ISSUE THREAD

**Issue by naveenkumarganesan** [TuvaLabs/markdown-delight-editor#4]
The editor area is not working properly when preview area is opened. Having problems for the following operations.

1) Text selection
2) Cursor positioning

**Comment by jpatel3** [comment]
@naveenkumarganesan Yes. I was able to reproduce this, while moving the cursor on using keyboard in editor area when preview is on. 

Also, another intermittent issue I noticed was often preview screen was loading blank. As soon as I opened inspect to check if any script error, it showed up fine.
